qa/tasks/cephfs/test_misc.py

```diff
@@ -216,6 +216,20 @@ def test_fs_lsflags(self):
         self.assertEqual(lsflags["allow_multimds_snaps"], True)
         self.assertEqual(lsflags["allow_standby_replay"], True)
 
+    def test_dispatch_queue_throttle_message(self):
+        """
+        That cluster log a warning when the Dispatch Queue Throttle Limit hits
+        """
+        self.config_set('global', 'ms_dispatch_throttle_bytes', 500)
```

Comment:
```suggestion
        self.config_set('mds', 'ms_dispatch_throttle_bytes', 500)
```

does this not work too?

Reply:
Will check that during testing. But why setting on `mds`, when `ms_dispatch_throttle_bytes` is a global config?